### PR TITLE
Get URL - Update Help

### PR DESCRIPTION
**Pull request by jmcadams-r7**
## Proposed Changes

This PR updates the help docs for a couple of typos and missing key features.

### CODE REVIEW

File: get_url/help.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ To reduce the number of subsequent requests the Etag and If-Modified-Since field
 
 # Requirements
 
-* Example: Requires an API Key from the product
-* Example: API must be enabled on the Settings page in the product
+* Get the contents of a web page
```
Comment (hwilsey-r7) [inline get_url/help.md]: Should this be under Key Features instead of Requirements?

Comment (jmcadams-r7) [reply get_url/help.md]: Yes. Fixed.

File: get_url/help.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
 # Description
 
-The Get URL InsighConnect plugin allows you to download files from a URL. Supported protocols are HTTP, HTTPS, and FTP.
+The Get URL InsightConnect plugin allows you to download files from a URL. Supported protocols are HTTP, HTTPS, and FTP.
```
Comment (jschipp-r7) [inline get_url/help.md]: We don't say InsightConnect in other places and it's a label on the Hub page.

```suggestion
The Get URL plugin allows you to download files from a URL. Supported protocols are HTTP, HTTPS, and FTP.
```

Comment (jmcadams-r7) [reply get_url/help.md]: fixed

**Pull request merged**